common/character_interactions/00_blackmail_interactions.txt

```diff
@@ -12,7 +12,8 @@
 	is_shown = {
```

Comment:
I feel like we should adapt these too.
<details>
<summary>Click to expand</summary>

![Uploading image.png…]()

</details>